stable/wordpress/templates/tls-secrets.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.ingress.enabled }}
+{{- range .Values.ingress.secrets }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .name }}
+  labels:
+    app: {{ template "fullname" $ }}
+    chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
+    release: "{{ $.Release.Name }}"
+    heritage: "{{ $.Release.Service }}"
+type: kubernetes.io/tls
+data:
+  tls.crt: {{ .certificate }}
```

Comment:
if you pipe this to ` | b64enc` you can remove the base64 requirement (http://masterminds.github.io/sprig/encoding.html)

Reply:
Done - thanks!